src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -355,23 +361,19 @@ void Camera::initialize_ray(
     ray.m_flags = VisibilityFlags::CameraRay;
     ray.m_depth = 0;
 
-    if (m_shutter_open_time == m_shutter_close_time)
-    {
-        ray.m_time =
-            ShadingRay::Time::create_with_normalized_time(
-                0.0f,
-                m_shutter_open_time,
-                m_shutter_close_time);
-    }
-    else
+    float sample_time = 0.0f;
```

Comment:
Small question : is it better make `sample_time` and pass it to `create_with_normalized_time()`, than pass `0.0` directly to function?

Reply:
Code-wise it is cleaner to have only one call instead of two which shared almost all the parameters.
I don't think there will be any significant difference in terms of speed.

Reply:
Um.. my question was compare about `create_with_norm_time(0.0, open_time, close_time)` and `create_with_norm_time(sample_time, open_time, close_time)`, very minor.

And I think this variable can be `const`.